.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
Unrelated change, should be removed from this commit.

Reply:
not unrelated, but something I would like to introduce as well :)

Reply:
Introduce in a separate pull request then.